files-community/Files
# Feature: Open all items in a tag

**Resolved / Related Issues**
   Closes #7493

**Validation**
How did you test these changes?
   1. Open app and navigate to `Home`
   2. Enable the `Tags` widget
   3. Right click on the title bar
   4. Click on `Open all`

**Screenshots (optional)**
![Screenshot 2023-07-18 224854](https://github.com/files-community/Files/assets/102259289/0bed76fc-5b35-482d-87d2-18622ddf1295)
